### PR TITLE
fix(*): fighting lint errors in code

### PR DESCRIPTION
**Pull request by kdinev**
I've gone through all non-grid, non-dv code blocks in the component documentation. Grids will be next.

### Checklist:
 
 - [ ] check topic's TOC/menu and paragraph headings
 - [ ] link to other topics using `../relative/path.md`
 - [ ] at the References section at the end of the topic add links to topics, samples, etc
 - [ ] reference API documentation instead of adding a section with API

<br />

 - [ ] use valid component names - [Data] Grid, `IgxSelectComponent`, `<igx-combo>`
 - [ ] use spell checker tool (VS Code, Grammarly, Microsoft Editor)
 - [ ] add inline `code blocks` for the names of classes / tags / properties
 - [ ] add language descriptor for the ```code blocks```
 - [ ] check broken links (use browser add-on)
 - [ ] check if sample is working and fully visible in the topic
 - [ ] check if sample is working and fully visible in the StackBlitz
 - [ ] check if code blocks match the code in StackBlitz demo

<br />

 - [ ] follow SEO guidelines to fill topic's metadata ([SEO guidelines](https://infragisticsinc297.sharepoint.com/:w:/g/Groups/marketing/EWz9InT4FDlErHCumxsKGY4Bd8H03yhRWxDFk47luRz-_Q?e=S5wWcx))

<br />

 - [ ] do not resolve requested changes (leave that to the reviewer)
 - [ ] add `pending-localization` label when the review of the PR is done
 - [ ] add a member from the localization team to translate it

### CODE REVIEW

File: jp/components/grids_templates/selection.md [hunk 1]
```diff
@@ -202,7 +199,7 @@ IgxGrid はコピーされたデータを取得し、コンテナ要素に貼り
 
 ## 既知の問題と制限
 
-- IE11 で選択を有効にした @@igComponent コンポーネントを使用するには、Angular アプリケーションの polyfill.ts に配列ポリフィルを明示的にインポートする必要があります。
+- IE11 で選択を有効にした @@igComponent コンポーネントを使用するには、Angular アプリケーションの polyfill.ts に配列ポリフィルを明示的にインポートする必要があります。 IE11 is no longer supported as of version 13.0.0.
```
Comment (ChronosSF) [inline jp/components/grids_templates/selection.md:202]: @randriova , added a pending localization tag for this sentence.

**Review by ChronosSF — DISMISSED** [review]

File: en/components/stepper.md [hunk 2]
```diff
@@ -253,7 +253,7 @@ Setting `none` to the both animation type inputs disables stepper animations.
 
 ## Keyboard Navigation
 
-Keyboard navigation in the Angular Material Stepper provides a rich variety of keyboard interactions to the end-user. This functionality is enabled by default and allows end-users to easily navigate through the steps. 
+The Angular Material Stepper provides a rich variety of keyboard interactions to the end-user. This functionality is enabled by default and allows end-users to easily navigate through the steps.
```
[the remark this answers is not in the document]
Comment (kdinev) [reply en/components/stepper.md:256]: Just suggest the change and I'll commit it 😄

[the remark this answers is not in the document]
Comment (ChronosSF) [reply en/components/stepper.md:256]:
```suggestion
Angular Material Stepper provides a rich variety of keyboard interactions to the end-user. This functionality is enabled by default and allows end-users to easily navigate through the steps. 
```

**Review by ChronosSF — APPROVED** [review]

File: jp/components/grids_templates/selection.md (hunk 1 above)
Comment (ChronosSF) [reply jp/components/grids_templates/selection.md:202]: @randriova , actually with the latest commits there are some topic titles that might need changing in jp as well.

**Review by ChronosSF — APPROVED** [review]

**Pull request merged**